src/services/completions.ts

```diff
@@ -88,10 +88,11 @@ namespace ts.Completions {
             if (pos === position) {
                 return;
             }
+            const realName = unescapeIdentifier(name);

```

Comment:
Can't. `name` is an EscapedIdentifier`, while `realName` needs to be a `string`. and the two are wholly incompatible.